fabric-sound-api-v1/src/testmod/java/net/fabricmc/fabric/test/sound/client/SineStream.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.test.sound.client;
+
+import java.nio.ByteBuffer;
+
+import javax.sound.sampled.AudioFormat;
```

Comment:
Just making sure, this won't cause a mess in macOS right?

Reply:
I don't have a Mac to test, but I'd be surprised if it doesn't work. It's just part of the standard JRE (`java.desktop` module, same as Swing) and given that vanilla MC uses it, I feel we'd have seen issues much earlier.

Reply:
Just tested it on my ARM mac and this works as expected. I dont think there is any concern here as its not an AWT/swing class. The vanilla usage proves this is a none issue.